api/app/views/mno_enterprise/jpi/v1/app_instances/_resource.json.jbuilder

```diff
@@ -8,6 +8,15 @@ json.oauth_keys_valid app_instance.oauth_keys_valid
 #json.microsoft_trial_url app_instance.microsoft_trial_url
 json.created_at app_instance.created_at
 
+if app_instance.per_user_licence?
+	json.per_user_licence app_instance.per_user_licence
+	json.licences_count app_instance.active_licences_count
+end
+
+if app_instance.under_free_trial?
+  json.free_trial_end_at time_ago_in_words(app_instance.free_trial_end_at)
```

Comment:
Hi Eduardo,

I think you could you try a mix of angular-translate variable replacement (https://angular-translate.github.io/docs/#/guide/06_variable-replacement) and momentjs i18n to use the right format depending of the language used.